### PR TITLE
Package.replacedBy on admin UI + infobox

### PR DESCRIPTION
**Pull request by isoos**
- Added further checks + tests: do not specify self, and do not specify another discontinued package.
- Minimal admin UI:
  Screenshot 2020-10-14 at 12 59 53

- Minimal exposure on InfoBox:
  Screenshot 2020-10-14 at 12 59 45

### CODE REVIEW

File: app/lib/frontend/templates/views/pkg/admin_page.mustache [hunk 1]
```diff
@@ -75,19 +75,36 @@
   <label for="-admin-is-discontinued-checkbox">Mark "discontinued"</label>
 </div>
 
+{{#show_replaced_by}}
+<h3>Replaced by</h3>
```
Comment (jonasfj) [inline app/lib/frontend/templates/views/pkg/admin_page.mustache:79]: Can we call this `suggested replacement`

File: app/lib/frontend/templates/views/pkg/info_box.mustache [hunk 2]
```diff
@@ -4,6 +4,13 @@
 
 {{& labeled_scores_html }}
 
+{{#replaced_by}}
+<h3 class="title">Replaced by</h3>
```
Comment (jonasfj) [inline app/lib/frontend/templates/views/pkg/info_box.mustache:8]: "suggested replacement" is probably a better word.

File: app/lib/frontend/templates/views/pkg/admin_page.mustache [hunk 3]
```diff
@@ -75,19 +75,36 @@
...
+<div class="-pub-form-row">
```
Comment (jonasfj) [inline app/lib/frontend/templates/views/pkg/admin_page.mustache:80]: We need a bit of text here along the lines of:

```markdown
When a package is <i>discontinued</i> the author may designate a <i>suggested replacement package</i>.
Package users will be suggested to consider using the designated replacement package.

Designating a replacement package is optional, and only serves to guide existing package users.
```

**Review by jonasfj — COMMENTED** [review]
@mit-mit, any thoughts?

**Comment by isoos** [comment]
Updated texts and screenshots, PTAL.

File: app/lib/frontend/templates/views/pkg/info_box.mustache [hunk 4]
```diff
@@ -4,6 +4,13 @@
 
 {{& labeled_scores_html }}
 
+{{#replaced_by}}
+<h3 class="title">Suggested replacement</h3>
+<p>
+  <a href="{{& replaced_by_link }}">{{ replaced_by }}</a>
```
Comment (jonasfj) [inline app/lib/frontend/templates/views/pkg/info_box.mustache:10]: Can we give this a link-title? (tooltip).

```suggestion
  <a href="{{& replaced_by_link }}" title="This package is discontinued, but author has suggested package:{{ replaced_by }} as a replacement">{{ replaced_by }}</a>
```

**Review by jonasfj — COMMENTED** [review]
LGTM,

**Comment by mit-mit** [comment]
LGTM

**Review by jonasfj — APPROVED** [review]

**Pull request merged**